Library/Homebrew/cmd/autoremove.rb

```diff
@@ -0,0 +1,53 @@
+# typed: false
+# frozen_string_literal: true
+
+require "formula"
+require "cli/parser"
+require "uninstall"
+
+module Homebrew
+  module_function
+
+  def autoremove_args
+    Homebrew::CLI::Parser.new do
+      usage_banner <<~EOS
+        `autoremove` [<options>]
+
+        Remove packages that weren't installed on request and are no longer needed.
```

Comment:
```suggestion
        Uninstall formulae that were only installed as a dependency of another formula and are now no longer needed.
```

* we call it "formula" rather than "package"
* avoid "install on request" jargon in favor of a more verbose explanation